src/schema-processor/jsonld/parser.ts

```diff
@@ -0,0 +1,69 @@
+import * as jsonld from 'jsonld/lib';
+import * as ldcontext from 'jsonld/lib/context';
+
+/**
+ * LDParser can parse JSONLD schema according to specification
+ *
+ * @export
+ * @beta
+ * @class LDParser
+ */
+export class LDParser {
+  /**
+   * ExtractTerms returns the terms definitions from the JSON-LD context
+   *
+   * @param {string} context - JSONLD context
+   * @returns Promise<Map<string, string>>
+   */
+  public static async extractTerms(context: string): Promise<Map<string, string>> {
+    let data;
+    try {
+      data = JSON.parse(context);
+    } catch (e) {
```

Comment:
include in try/catch block async call also

Reply:
done